bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/validation/internal/ConfigDescriptionParameterValidator.java

```diff
@@ -12,20 +12,30 @@
 
 /**
  * The {@link ConfigDescriptionParameterValidator} can be implemented to provide a specific validation of a
- * {@link ConfigDescriptionParameter} and its actual value.
+ * {@link ConfigDescriptionParameter} and its value to be set.
  *
  * @author Thomas Höfer - Initial contribution
  */
 public interface ConfigDescriptionParameterValidator {
 
     /**
+     * Returns true, if the validator is to be executed for the given parameter and its given value, otherwise false.
+     *
+     * @param parameter the corresponding config description parameter
+     * @param value the value to be set for the config description parameter
+     *
+     * @return true, if the validator is to be executed for the given parameter and its given value, otherwise false
+     */
+    boolean isExecutable(ConfigDescriptionParameter parameter, Object value);
```

Comment:
I agree... I just saw that there is no valiation in the validate operation of the RequiredValidator... that should be changed...

Do you want me to remove the isExecutabel / isApplicable operation?